packages/webview_flutter/webview_flutter_wkwebview/lib/src/v4/src/webkit_webview_controller.dart

```diff
@@ -306,7 +312,7 @@ class WebKitWebViewController extends PlatformWebViewController {
   Future<void> setPlatformNavigationDelegate(
     covariant WebKitNavigationDelegate handler,
   ) {
-    _onProgress = handler._onProgress;
+    _getOnProgressCallback = () => handler._onProgress;
```

Comment:
Per offline discussion, let's keep `handler` around and use that directly, instead of having this function getter, as I think that will be easier to understand.